includes/admin/class.llms.admin.setup.wizard.php

```diff
@@ -123,90 +146,114 @@ public function generator_course_status( $status ) {
 	 * @return string
 	 */
 	public function get_current_step() {
-		return empty( $_GET['step'] ) ? 'intro' : llms_filter_input( INPUT_GET, 'step', FILTER_SANITIZE_STRING );
+		return empty( $_GET['step'] ) ? 'intro' : llms_filter_input( INPUT_GET, 'step', FILTER_SANITIZE_STRING ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 	}
 
 	/**
 	 * Get slug if next step
 	 *
-	 * @since  3.0.0
+	 * @since 3.0.0
+	 * @since [version] Combined combined if/elseif into a single condition & use strict `array_search()` comparison.
 	 *
 	 * @param string $step Step to use as current.
 	 * @return string|false
 	 */
 	public function get_next_step( $step = '' ) {
-		if ( ! $step ) {
-			$step = $this->get_current_step();
-		}
-		$steps = $this->get_steps();
-		$keys  = array_keys( $steps );
-		$i     = array_search( $step, $keys );
-		if ( false === $i ) {
-			return false;
-		} elseif ( $i++ >= count( $keys ) - 1 ) {
+
+		$step = $step ? $step : $this->get_current_step();
+		$keys = array_keys( $this->get_steps() );
+		$i    = array_search( $step, $keys, true );
+
+		// Next step doesn't exist or the next step would be greater than the index of the last step.
+		if ( false === $i || $i++ >= count( $keys ) - 1 ) {
 			return false;
-		} else {
-			return $keys[ $i++ ];
 		}
+
+		return $keys[ $i++ ];
```

Comment:
golf
```suggestion
		return $keys[ $i ];
```
because: http://sandbox.onlinephpfunctions.com/code/3ddf7183a2773398e4d7ea2661fd60c00d011d4d

Reply:
FASCINATING. I didn't know this.

However, I think now that I understand it the code feels more readable as:

```php

		if ( false === $i || $i + 1 >= count( $keys ) - 1 ) {
			return false;
		}

		return $keys[ ++$i ];

```

Would you agree?

Reply:
I agree <3